src/lib/phdi-transforms/phdi_transforms/basic.py

```diff
@@ -0,0 +1,26 @@
+def transform_name(raw: str) -> str:
+    """trim spaces, capitalize, etc
```

Comment:
nit: We're converting to uppercase, not capitalizing.

Reply:
Fair enough. We can change this.